pyaedt/modules/Boundary.py

```diff
@@ -356,47 +356,191 @@ def delete(self):
 
 def disable_auto_update(func):
     def wrapper(self, *args, **kwargs):
-        auto_update = self.auto_update
-        self.auto_update = False
+        obj = self
+        if not hasattr(self, "auto_update"):
+            obj = self.pcb
+        auto_update = obj.auto_update
+        obj.auto_update = False
         out = func(self, *args, **kwargs)
-        self.update()
-        self.auto_update = auto_update
+        if auto_update:
+            obj.update()
+        obj.auto_update = auto_update
         return out
 
     return wrapper
 
 
-class NativeComponentPCB(NativeComponentObject, object):
-    """Manages Native Component PCB data and execution.
+class PCBSettingsPackageParts(object):
+    def __init__(self, pcb_obj, app):
+        self._app = app
+        self.pcb = pcb_obj
+        self._solderbumps_map = {"Lumped": "SbLumped", "Cylinders": "SbCylinder", "Boxes": "SbBlock"}
+
+    def __eq__(self, other):
+        if isinstance(other, str):
+            return other == "Package"
+        elif isinstance(other, self.__class__):
+            return self.__dict__ == other.__dict_
+        else:
+            return False
 
-    Parameters
-    ----------
-    app : object
-        AEDT application from the ``pyaedt.application`` class.
-    component_type : str
-        Type of the component.
-    component_name : str
-        Name of the component.
-    props : dict
-        Properties of the boundary.
+    def __ne__(self, other):
+        return not self.__eq__(other)
 
-    Examples
-    --------
-    In this example, the returned object, ``par_beam`` is a ``pyaedt.modules.Boundary.NativeComponentObject`` instance.
-    >>> from pyaedt import Icepak
-    >>> ipk = Icepak(solution_type="SBR+")
-    >>> par_beam = ipk.create_ipk_3dcomponent_pcb()
-    """
+    @pyaedt_function_handler()
+    @disable_auto_update
+    def set_solderballs_modeling(self, modelling=None):
+        """Set how to model solderballs.
 
-    def __init__(self, app, component_type, component_name, props):
-        NativeComponentObject.__init__(self, app, component_type, component_name, props)
+        Parameters
+        ----------
+        modelling : str, optional
```

Comment:
Modeling is the preferred spelling in American English. Should we be using it instead?